### PR TITLE
4.3

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/remove [hunk 1]
```diff
@@ -28,6 +28,14 @@ ynh_script_progression --message="Removing app main directory..." --weight=1
...
 ynh_secure_remove --file="$final_path"
 
+#=================================================
+# REMOVE DEPENDENCIES
+#=================================================
+ynh_script_progression --message="Removing dependencies..." --weight=1
+
+# Remove metapackage and its dependencies
+ynh_remove_app_dependencies
+
```
Comment (alexAubin) [inline scripts/remove:38]: Same as for yellowcms: CI is failing here, maybe because we should first remove the PHP conf

**Comment by ericgaspar** [comment]
!tastemetitus

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**